packetizer.go

```diff
@@ -68,5 +88,27 @@ func (p *packetizer) Packetize(payload []byte, samples uint32) []*Packet {
 	}
 	p.Timestamp += samples
 
+	if p.AbsSendTime != 0 {
+		t := toNtpTime(time.Now()) >> 14
+		//apply http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time
+		packets[len(payloads)-1].Header.Extension = true
+		packets[len(payloads)-1].ExtensionProfile = 0xBEDE
+		packets[len(payloads)-1].ExtensionPayload = []byte{
+			//the first byte is
+			// 0 1 2 3 4 5 6 7
+			//+-+-+-+-+-+-+-+-+
+			//|  ID   |  len  |
+			//+-+-+-+-+-+-+-+-+
+			//per RFC65285
+			//Len is the number of bytes in the extension - 1
+
+			byte((p.AbsSendTime << 4) | 2),
```

Comment:
I'm really confused about what `p.AbsSendTime` represents. Can you choose a different name that is more descriptive?

Reply:
it's the rtp extension number. I wonder if it should be a sub-structure -- `ExtNumbers.AbsSendTime`?

Reply:
I liked the flat version better, personally. Sub-structure might make sense if we supported more extensions, but at that point an array makes more sense.